lib/logstash/outputs/influxdb.rb

```diff
@@ -69,6 +72,19 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   #
   config :coerce_values, :validate => :hash, :default => {}
 
+  # Automatically use fields from the event as the data points sent to Influxdb
+  config :use_event_fields_for_data_points, :validate => :boolean, :default => false
```

Comment:
Feels like this setting could use a more detailed explanation. Can you describe the behavior more?